lib/rspec-rails.rb

```diff
@@ -8,6 +8,12 @@ module RSpec
   module Rails
     # Railtie to hook into Rails.
     class Railtie < ::Rails::Railtie
+
+      # As of Rails 5.1.0 you can register directories to work with `rake notes`
+      if Gem::Requirement.new(">= 5.1.0").satisfied_by?(Gem::Version.new(::Rails.version))
```

Comment:
Could you switch this away from using rubygems please? It's generally a good idea to not assume it's present, a check on the rails version string is sufficient normally (e.g. ::Rails.version >= "5.1.0").